x/bridge/keeper/keeper.go

```diff
@@ -0,0 +1,60 @@
+package keeper
+
+import (
+	"fmt"
+
+	"github.com/cometbft/cometbft/libs/log"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+
+	"github.com/osmosis-labs/osmosis/v23/x/bridge/types"
+)
+
+type Keeper struct {
+	storeKey   storetypes.StoreKey
+	paramSpace paramtypes.Subspace
+
+	accountKeeper      types.AccountKeeper
+	tokenFactoryKeeper types.TokenFactoryKeeper
+
+	govModuleAddr string
+}
+
+// NewKeeper returns a new instance of the x/bridge keeper.
+func NewKeeper(
+	storeKey storetypes.StoreKey,
+	paramSpace paramtypes.Subspace,
+	accountKeeper types.AccountKeeper,
+	tokenFactoryKeeper types.TokenFactoryKeeper,
+	govModuleAddr string,
+) Keeper {
+	// ensure bridge module account is set
+	if addr := accountKeeper.GetModuleAddress(types.ModuleName); addr == nil {
+		panic("the bridge module account has not been set")
+	}
+
+	if !paramSpace.HasKeyTable() {
+		paramSpace = paramSpace.WithKeyTable(types.ParamKeyTable())
+	}
+
+	return Keeper{
+		storeKey:           storeKey,
+		paramSpace:         paramSpace,
+		accountKeeper:      accountKeeper,
+		tokenFactoryKeeper: tokenFactoryKeeper,
+		govModuleAddr:      govModuleAddr,
+	}
+}
+
+// Logger returns a logger for the x/bridge module.
+func (k Keeper) Logger(ctx sdk.Context) log.Logger {
+	return ctx.Logger().With("module", fmt.Sprintf("x/%s", types.ModuleName))
+}
+
+// CreateModuleAccount creates a module account with minting and burning capabilities.
+// This account isn't intended to store any coins, it purely mints and burns them
+// on behalf of the admin of respective denoms, and sends to the relevant address.
+func (k Keeper) CreateModuleAccount(ctx sdk.Context) {
+	k.accountKeeper.GetModuleAccount(ctx, types.ModuleName)
+}
```

Comment:
See previous comment, I might be wrong but I think we just need to initialize this here:

https://github.com/osmosis-labs/osmosis/blob/520973d8326a90d21dcd461212f90f549ee7b001/app/modules.go#L107-L135

Reply:
Will double-check it in the wiring pr then